src/core/blockchain/zilliqa/stats.ts

```diff
@@ -25,7 +25,7 @@ export class Stats extends GenericStats<Client> {
             this.client.chainId.toString()
         );
 
-        let availableToDelegate = new BigNumber(data.balance.unstaked);
+        let availableToDelegate = new BigNumber(data.balance.unstaked || 0);
```

Comment:
In the last pr you had 3 places with this piece of code, so we will have this issue there also.